src/frontend/handlers.go

```diff
@@ -72,6 +72,12 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
+	if os.Getenv("CONVERT_CURRENCIES") == "Active" {
```

Comment:
Can you add a comment here? This will be confusing to users not familiar with the recipes

Reply:
Added a comment but didn't want to expose that this is part of a recipe so users using the recipe won't discover the "fix" too easily. Let me know if you think it needs more

Reply:
Ok, yeah that makes sense! looks good then